content/applications/hr/employees/new_employee.rst

```diff
@@ -103,25 +113,27 @@ following information for each entry.
    :align: center
    :alt: A resumé entry form with all the information populated.
 
-- :guilabel:`Title`: Type in the title of the previous work experience.
-- :guilabel:`Type`: From the drop-down menu, select either :guilabel:`Experience`,
+- :guilabel:`Title`: type in the title of the previous work experience.
+- :guilabel:`Employee`: select the employee from the drop-down menu.
+- :guilabel:`Type`: from the drop-down menu, select either :guilabel:`Experience`,
   :guilabel:`Education`, :guilabel:`Side Projects`, :guilabel:`Internal Certification`,
   :guilabel:`Completed Internal Training`, or type in a new entry, then click :guilabel:`Create
-  "(new entry)"`.
-- :guilabel:`Display Type`: Select either :guilabel:`Classic`, :guilabel:`Certification`, or
-  :guilabel:`Course` from the drop-down menu.
-- :guilabel:`Date Start` and :guilabel:`Date End`: Enter the start and end dates for the work
-  experience. To select a date, use the :guilabel:`< (left)` and :guilabel:`> (right)` arrow icons
-  to scroll to the desired month, then click on the day to select it.
-- :guilabel:`Description`: Enter any relevant details in the field.
+  "(Type)"`.
+- :guilabel:`Display Type`: From the drop-down menu, choose :guilabel:`Classic` for typical work
+  experience, :guilabel:`Certification` for experience gained through a certificate, or
+  :guilabel:`Course` for non-certified classes.
+- :guilabel:`Duration`: enter the start and end dates for the work experience. To select a date, use
+  the :guilabel:`< (left arrow)` and :guilabel:`> (right arrow)` icons to scroll to the desired
+  month, then click on the day to select it.
+- :guilabel:`Description`: enter any relevant details in the field.
 
 Once all the information is entered, click the :guilabel:`Save & Close` button if there is only one
 entry to add, or click the :guilabel:`Save & New` button to save the current entry and create
 another resumé line.
 
 .. note::
    After the new employee form is saved, the current position and company is automatically added to
-   the :guilabel:`Resumé` tab, with the end date listed as *current*.
+   the :guilabel:`Resumé` tab, with the end date listed as `current`.
 
 Skills
```

Comment:
Same for the previous comment- Zac had made the original change with the accent, so I did not want to remove it. I think it might be best to stay consistent, even though the buttons do not have the accent. I will leave this for him to decide in the final review (or whoever is doing the final review).